command/plan.go

```diff
@@ -13,13 +13,15 @@ import (
 // migration operations to a temporary state.
 type PlanCommand struct {
 	Meta
-	out string
+	backendConfig []string
```

Comment:
The tfmigrate apply command also uses the same logic, so we need to add this flag to the tfmigrate apply command too.

Reply:
Add --backend-config to `tfmigrate apply` in 6cb4eca5d0724e4412416cb06fb4e22765769e75